.travis.yml

```diff
@@ -74,7 +78,7 @@ before_install:
   # On stable PHPCS versions, allow for PHP deprecation notices.
   # Unit tests don't need to fail on those for stable releases where those issues won't get fixed anymore.
   - |
-    if [[ "${TRAVIS_BUILD_STAGE_NAME^}" != "Sniff" && $PHPCS_BRANCH != "dev-master" && $WPCS_BRANCH != "dev-develop" ]]; then
```

Comment:
This line should not be changes. As per the comment above it, deprecation notices should only be allowed on _unstable_ builds. For WPCS `dev-master` is _stable_, while `dev-develop` is _unstable_.